packages/controller/package.json

```diff
@@ -60,17 +60,19 @@
     "exports": "named"
   },
   "peerDependencies": {
-    "starknet": "catalog:",
-    "open": "^10.1.0"
+    "open": "^10.1.0",
+    "starknet": "catalog:"
   },
   "dependencies": {
     "@cartridge/account-wasm": "workspace:*",
     "@cartridge/penpal": "catalog:",
+    "@metamask/sdk": "^0.32.1",
+    "@phantom/wallet-sdk": "^0.0.23",
     "@starknet-io/types-js": "catalog:",
     "@telegram-apps/sdk": "^2.4.0",
-    "base64url": "catalog:",
     "cbor-x": "^1.5.0",
-    "fast-deep-equal": "catalog:"
+    "fast-deep-equal": "catalog:",
+    "starknetkit": "^2.6.1"
```

Comment:
I wonder if most of these should be peer deps since the calling app will likely have them already included